app/assets/javascripts/modal_login.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
+  $("a.sign_in").on("click", function(){ // show login form
+    $('#log_in_modal').modal('show');
+  });
+  $("a.sign_up").on("click", function(){ // hide login form, show signup form
+    $('#log_in_modal').modal('hide');
+    $('#signup_modal').modal('show');
```

Comment:
Nhung ơi, chỗ này em phải viết js để tắt hết tất cả các modal trước khi mở 1 modal ra nhé

Reply:
$('#log_in_modal').modal('hide');
dòng này em tắt các modal trước rồi ạ

Reply:
trường hợp em mở modal sign_up trước thì sao??

Reply:
![login](https://user-images.githubusercontent.com/39556344/53622443-67d8cd80-3c2c-11e9-9f52-02fffa9567ad.gif)
em dùng $($(this).closest('.modal')).modal('hide'); để tắt cái modal đang mở được k ạ.